requirements_production.txt

```diff
@@ -1,10 +1,11 @@
 # Requirements for OpenSlides in production in alphabetical order
-bleach>=1.5.0,<1.6
-channels>=1.1,<1.2
+bleach>=1.5.0,<2.2
+channels<1.2
```

Comment:
Please keep the lower version as our minimal supported version: `channels>=1.1,<1.2`

Reply:
adjusted this back to the original state.